src/pages/tezos/customizebot.js

```diff
@@ -134,7 +134,7 @@ const Bot = ({
   shininess,
 }) => {
   const group = useRef();
-  const { scene } = useGLTF('/compressedv5.glb');
+  const { scene } = useGLTF('/compressed.glb');
```

Comment:
@harsh242 This old version is working. Please try to debug why is that's the case? I am stuck here.